src/libraries/Common/src/System/SR.cs

```diff
@@ -41,6 +41,10 @@ internal static string GetResourceString(string resourceKey, string? defaultStri
             return resourceString!; // only null if missing resources
         }
 
+#if !SYSTEM_PRIVATE_CORELIB
+        private static string? InternalGetResourceString(string resourceKey) => ResourceManager.GetString(resourceKey);
+#endif
```

Comment:
Instead of adding this extra method, could we make the earlier line be:
```C#
resourceString =
#if SYSTEM_PRIVATE_CORELIB
    InternalGetResourceString(resourceKey);
#else
    ResourceManager.GetString(resourceKey);
#endif
```
?